experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts

```diff
@@ -14,30 +14,24 @@ export class Bubblebench extends DataObject {
 	}
 	private maybeTree?: ISharedTree = undefined;
 	private maybeAppState?: AppState = undefined;
-	static treeFactory = new SharedTreeFactory();
```

Comment:
There is no need to cache the `SharedTreeFactory` instance.

For comparison, see the implementations of `SharedFoo.getFactory()`, which returns `new SharedFooFactory()` (uncached).